### DISCUSSION TITLE
Select profile in config file

### DISCUSSION THREAD

**Discussion by bryanburke** [ansible/ansible-lint#2272] [Ideas]
I believe the ability to select the rule profile within the `.ansible-lint` or `.config/ansible-lint.yml` config file would be helpful. For example, this YAML key-value pair:

```yaml
profile: production
```

Would be equivalent to this command-line argument:

```sh
ansible-lint --profile=production # or -P=production
```

The command-line argument would of course take precedence. Also, if selecting multiple profiles is possible, then a YAML array may be more appropriate:

```yaml
profiles:
  - production
```

Thanks!

**Comment by ssbarnea** [comment]
We will add this, I only wanted to keep the original PR that added support for profiles as a "MVP".
